<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LearnHeatingValve</class>
 <widget class="QWidget" name="LearnHeatingValve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <widget class="QPushButton" name="teachDescription">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Please press the LEARN
button on the heating valve!</string>
   </property>
  </widget>
  <widget class="QLabel" name="sensorLearnSuccess">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>320</width>
     <height>44</height>
    </rect>
   </property>
   <property name="text">
    <string>Heating valve identified!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="settingsHead" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="settingLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>320</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Heating Valve</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/settings/icons/arrow_left_white.svg</normaloff>:/settings/icons/arrow_left_white.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelTemperature">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Temperature:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelValvePosition">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Valve Position:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTemperatureValue">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>80</y>
     <width>55</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTemperatureUnit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"> Â°C</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelValvePositionValue">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>120</y>
     <width>55</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="removeValves">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>45</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove all Heating Valves from Room</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
